src/regexes.ts

```diff
@@ -0,0 +1,143 @@
+// https://www.regular-expressions.info/unicode.html
+import * as XRegExp from 'xregexp'
+
+// scripts
+export const common = XRegExp('\\p{Common}')
+export const arabic = XRegExp('\\p{Arabic}')
+export const armenian = XRegExp('\\p{Armenian}')
+export const bengali = XRegExp('\\p{Bengali}')
+export const bopomofo = XRegExp('\\p{Bopomofo}')
+export const braille = XRegExp('\\p{Braille}')
+export const buhid = XRegExp('\\p{Buhid}')
+export const canadianAboriginal = XRegExp('\\p{Canadian_Aboriginal}')
+export const cherokee = XRegExp('\\p{Cherokee}')
+export const cyrillic = XRegExp('\\p{Cyrillic}')
+export const devanagari = XRegExp('\\p{Devanagari}')
+export const ethiopic = XRegExp('\\p{Ethiopic}')
+export const georgian = XRegExp('\\p{Georgian}')
+export const greek = XRegExp('\\p{Greek}')
+export const gujarati = XRegExp('\\p{Gujarati}')
+export const gurmukhi = XRegExp('\\p{Gurmukhi}')
+export const han = XRegExp('\\p{Han}')
+export const hangul = XRegExp('\\p{Hangul}')
+export const hanunoo = XRegExp('\\p{Hanunoo}')
+export const hebrew = XRegExp('\\p{Hebrew}')
+export const hiragana = XRegExp('\\p{Hiragana}')
+export const inherited = XRegExp('\\p{Inherited}')
+export const kannada = XRegExp('\\p{Kannada}')
+export const katakana = XRegExp('\\p{Katakana}')
+export const khmer = XRegExp('\\p{Khmer}')
+export const lao = XRegExp('\\p{Lao}')
+export const latin = XRegExp('\\p{Latin}')
+export const limbu = XRegExp('\\p{Limbu}')
+export const malayalam = XRegExp('\\p{Malayalam}')
+export const mongolian = XRegExp('\\p{Mongolian}')
+export const myanmar = XRegExp('\\p{Myanmar}')
+export const ogham = XRegExp('\\p{Ogham}')
+export const oriya = XRegExp('\\p{Oriya}')
+export const runic = XRegExp('\\p{Runic}')
+export const sinhala = XRegExp('\\p{Sinhala}')
+export const syriac = XRegExp('\\p{Syriac}')
+export const tagalog = XRegExp('\\p{Tagalog}')
+export const tagbanwa = XRegExp('\\p{Tagbanwa}')
+export const taile = XRegExp('\\p{TaiLe}')
+export const tamil = XRegExp('\\p{Tamil}')
+export const telugu = XRegExp('\\p{Telugu}')
+export const thaana = XRegExp('\\p{Thaana}')
+export const thai = XRegExp('\\p{Thai}')
+export const tibetan = XRegExp('\\p{Tibetan}')
+export const yi = XRegExp('\\p{Yi}')
+
+export function getScripts(string: string): object {
+    const scripts = {

```

Comment:
Instead of making this large object here, you could have all of the above regexs in their own file, and import the entire file,
```ts
import * as scripts from './regexes' // a file containing all of your declared regexes above
```